_config/config.yml

```diff
@@ -4,17 +4,11 @@ Name: ss-seo
 SilverStripe\SiteConfig\SiteConfig:
   enable_seo_cache_lifetime: true
   extensions:
-    - CatchDesign\SS\SEO\Extensions\SSRobotsConfigExtension
+    - Catch\SS_SEO\SSRobotsConfigExtension
 
 SilverStripe\CMS\Model\SiteTree:
   extensions:
-    - CatchDesign\SS\SEO\Extensions\CanonicalExtension
-    - CatchDesign\SS\SEO\Extensions\SiteTreeRobotsExtension
+    - Catch\SS_SEO\CanonicalExtension
 
-SilverStripe\Core\Injector\Injector:
-  SilverStripe\Assets\Image_Backend:
-    properties:
-      Quality: 70
-  Psr\SimpleCache\CacheInterface.cacheblock:
-    constructor:
-      defaultLifetime: 2419200
+GDBackend:
```

Comment:
what is this change for ? https://docs.silverstripe.org/en/5/developer_guides/files/images/